docs/HOWTO.md

```diff
@@ -1601,6 +1602,36 @@ sudo apt install -y vim-gtk
 
 ---
 
+### Use a subdirectory of your dotfiles repo as the root of the source state
+
+By default, chezmoi uses the root of your dotfiles repo as the root of the
+source state. If your source state contains a lot of entries in its root, then
+your dotfiles repo will in turn contain a lot of entries in its root.
+
```

Comment:
It's probably a good idea to first explain the first solution, which would be to keep the `.chezmoiignore` updated. And also to mention why or where it would be a better idea to use `.chezmoiroot`. E.g.: with `.chezmoiroot`, if you add a new file to the root of the repository, you don't need to remember to manually add it to the `.chezmoiignore`.

Reply:
Done (in working copy).